code/modules/hivemind/self_defense.dm

```diff
@@ -134,7 +134,7 @@
 	for(var/i = 1 to amount)
 		var/turf/spawn_loc = pick(places_to_spawn)
 		champion = new champion_path(spawn_loc)
-		champion.maxHealth += 150
+		champion.adjustMaxHealth(150)
```

Comment:
You kinda used wrong function : setMaxHealth should be used, the next line can use a function adjustHealth

Reply:
But reading more this function, why we have adjust**Max**Health and setMaxHealth

Reply:
Okay, nevermind I found your newly added function. It turns out adjustMaxHealth hasnt never existed.